azure-py-aks/README.md

```diff
@@ -0,0 +1,67 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Azure Kubernetes Service (AKS) Cluster
+
+This example deploys an AKS cluster, virtual network and Azure Container Registry and grants AKS permissions to access and manage those.
+
+## Deploying the App
+
+To deploy your infrastructure, follow the below steps.
+
+### Prerequisites
+
+1. [Install Pulumi](https://pulumi.io/install)
+2. [Install Python 3.6](https://www.python.org/downloads/)
+3. [Configure Azure Credentials](https://pulumi.io/install/azure.html)
+4. [Generate SSH Key](https://git-scm.com/book/en/v2/Git-on-the-Server-Generating-Your-SSH-Public-Key)
+
+### Steps
+
+After cloning this repo, from this working directory, run these commands:
+
+1. Install the required Python packages packages:
+
+    ```bash
+    $ pip install -r requirements.txt
+    ```
+
+2. Create a new stack, which is an isolated deployment target for this example:
+
+    ```bash
+    $ pulumi stack init
+    ```
+
+3. Set the configuration variables for this program, they have default values, so you dont have to do that:
+
+    ```bash
+    $ pulumi config set prefix all_resources_will_be_prefixed_with_this_value
+    $ pulumi config set password service_principal_password
+    $ pulumi config set sshkey your_public_ssh_key
+    $ pulumi config set location any_valid_azure_location_for_aks
```

Comment:
While you're making changes, if you omit the `$` at the beginning of these code blocks, it makes it much easier to copy and paste blocks of code. We'd still have to change the values of the config variables, but at least you can copy+paste once instead of 4 times.

Reply:
thats quite the opposite of what I was asked in the other PR. I submitted it without `$` and was asked to add those back....

Reply:
Technically our style is to add `$` when there is output and omit it when there is not. If that's not what you were told, then that was wrong. :) This is is meant to conform with how ~all markdown linters.

Reply:
But, it's also not a point worth dying over, which is why I bring it up in an offhand way.

Reply:
https://github.com/pulumi/docs/pull/895

Reply:
cc @justinvp. See above.

Reply:
Let's not block on this since we're already inconsistent anyway.

Reply:
ok, i think I've resolved the other ones.